app/decorators/exercise_decorator.rb

```diff
@@ -3,9 +3,6 @@ class ExerciseDecorator < Draper::Decorator
 
   def status
     status = h.current_user.exercise_status_hash[object.id]
-    if (status.nil?)
-      status = "not started"
-    end
-    return status
+    status.present? ? status : status = "not started"
```

Comment:
So at what point what a status not be present? I thought you're setting the default to `0` for all exercises, would we ever have a case where it would return "not started"?